tests/storage/vmfactory.go

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ *
+ */
+
+package storage
+
+import (
+	virtv1 "kubevirt.io/api/core/v1"
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+
+	"kubevirt.io/kubevirt/pkg/libvmi"
+
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+func NewTinyOSVMWithDataVolume(dv *cdiv1.DataVolume, extraVMOpts ...libvmi.VMOption) *virtv1.VirtualMachine {
```

Comment:
What does the name `NewTinyOSVMWithDataVolume` stands for? What do you mean by "TinyOS"?

Reply:
I mean minimal linux distros like alpine/cirros